NakedJavaScript/mensa-projekt
# Add chart.js

# What has changed:
* Add chart.js for drawing charts (for the umsatz page).
* Add example js function
# How to test:
* Go to the umsatz page
* select an option under in the drop-down menu
* you should now see an example chart.
